lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -735,6 +738,27 @@ def test_text_search(self, text_search_rewrite):
             "text": ["test search string"],
         })
 
+    def test_following(self):
+        self.register_subscribed_threads_response(self.user, [], page=1, num_pages=1)
+        result = get_thread_list(
+            self.request,
+            self.course.id,
+            page=1,
+            page_size=11,
+            following=True,
+        )
+        self.assertEqual(
+            result,
+            {"results": [], "next": None, "previous": None, "text_search_rewrite": None}
+        )
+        self.assert_last_query_params({
+            "course_id": [unicode(self.course.id)],
+            "sort_key": ["date"],
+            "sort_order": ["desc"],
+            "page": ["1"],
+            "per_page": ["11"],
+        })
```

Comment:
ah, thanks.   I had confused the call to `get_thread_list` (from `api`) with `self.get_thread_list` which actually does register that endpoint.